test/NinjaTools.FluentMockServer.Tests/FluentAPI/Builders/FluentExpectationBuilderTests.cs

```diff
@@ -104,17 +92,21 @@ public void Should_Set_TimeToLive()
             var builder = new FluentExpectationBuilder();
             
             // Act
-            var result = builder
+            var expectation = builder
                 .OnHandlingAny()
                 .RespondWith(HttpStatusCode.OK)
                 .WhichIsValidFor(10)
-                .Setup()
-                .Expectations.First()
-                .AsJson();
-            
+                .Setup().Expectations.First();
+
+            Dump(expectation, "Expectation with TimeToLive", Serializer.Serialize);
+
             // Assert
-            _outputHelper.WriteLine(result);
-            result.Should().MatchRegex(@"(?m)\s*""timeToLive"":\s*\{\s*""timeUnit"":\s*""SECONDS""\s*,\s*""timeToLive"":\s*10\s*,\s*""unlimited""\s*:\s*false\s*}");
+            expectation.TimeToLive.TimeToLive.Should().Be(10);
+            expectation.TimeToLive.TimeUnit.Should().Be(TimeUnit.Seconds);
+            expectation.TimeToLive.Unlimited.Should().BeFalse();
+
+            // Assert
+          //  result.Should().MatchRegex(@"(?m)\s*""timeToLive"":\s*\{\s*""timeUnit"":\s*""SECONDS""\s*,\s*""timeToLive"":\s*10\s*,\s*""unlimited""\s*:\s*false\s*}");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4885942)